examples/image_embedder/android/app/src/main/java/com/google/mediapipe/examples/imageembedder/ImageEmbedderHelper.kt

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.mediapipe.examples.imageembedder
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.os.SystemClock
+import android.util.Log
+import com.google.mediapipe.framework.MediaPipeException
+import com.google.mediapipe.framework.image.BitmapImageBuilder
+import com.google.mediapipe.tasks.core.BaseOptions
+import com.google.mediapipe.tasks.core.Delegate
+import com.google.mediapipe.tasks.vision.imageembedder.ImageEmbedder
+import com.google.mediapipe.tasks.vision.imageembedder.ImageEmbedder.ImageEmbedderOptions
+
+class ImageEmbedderHelper(
+    private val context: Context,
+    var currentDelegate: Int = DELEGATE_CPU,
+    var currentModel: Int = MODEL_SMALL,
+    var listener: EmbedderListener?
+) {
+    private var imageEmbedder: ImageEmbedder? = null
+
+    init {
+        setupImageEmbedder()
+    }
+
+    fun setupImageEmbedder() {
+        val baseOptionsBuilder = BaseOptions.builder()
+        when (currentDelegate) {
+            DELEGATE_CPU -> {
+                baseOptionsBuilder.setDelegate(Delegate.CPU)
+            }
+            DELEGATE_GPU -> {
+                baseOptionsBuilder.setDelegate(Delegate.GPU)
+            }
+        }
+        when (currentModel) {
+            MODEL_SMALL -> {
+                baseOptionsBuilder.setModelAssetPath("mobilenet_v3_small.tflite")
```

Comment:
Let's move the model names into constants at the bottom of the file so they're a little more easily found by our devs.